Meta/HeaderCheck/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 execute_process(COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/generate_all.py" "${SERENITY_ARCH}" OUTPUT_VARIABLE SOURCES_STRING)
 string(REPLACE "\n" ";" SOURCES_LIST ${SOURCES_STRING})
 
+add_compile_options(-Wno-psabi)
```

Comment:
It'd be nice for the commit to explain why this is needed

Reply:
LibSoftGPU does not compile without it, as evidenced by `Userland/Libraries/LibSoftGPU/CMakeLists.txt:9`.

Reply:
Right, just saying that should be in the commit message

Reply:
Both commit message and a comment in CMakeLists.txt now explain what's going on.